ledger/src/blockstore_meta.rs

```diff
@@ -292,12 +304,11 @@ mod test {
         use ErasureMetaStatus::*;
 
         let set_index = 0;
-        let erasure_config = ErasureConfig::default();
+        let erasure_config = ErasureConfig::new(8, 16);
```

Comment:
nit: define 8/16 as const?

Reply:
it is just a test so it is probably fine either way